scheduler/docs/configuration.asc

```diff
@@ -98,6 +98,13 @@ We'll look at the configurable options in turn:
   This sets the role that Cook will connect to Mesos with. Default: `*`
   You can omit this property unless you've enabled security features in Mesos. The value should be in authorized list for the current `:principal` in `register_frameworks` Action in Mesos Authorization file. See http://mesos.apache.org/documentation/latest/authorization/ for details.
 
+`:enable-gpu-support`::
```

Comment:
Can you add a doc, not sure where is the best place describing the steps necessary to get gpus working with cook and mesos. I expect it will lean heavily on Kevin's gpu doc he posted recently I just want a single place to point people if they ask.